packages/view/backend/sirius-components-view-emf/src/main/java/org/eclipse/sirius/components/view/emf/form/ViewFormDescriptionConverter.java

```diff
@@ -72,7 +81,13 @@ public boolean canConvert(RepresentationDescription representationDescription) {
     @Override
     public IRepresentationDescription convert(RepresentationDescription representationDescription, List<RepresentationDescription> allRepresentationDescriptions, AQLInterpreter interpreter) {
         org.eclipse.sirius.components.view.FormDescription viewFormDescription = (org.eclipse.sirius.components.view.FormDescription) representationDescription;
-        ViewFormDescriptionConverterSwitch dispatcher = new ViewFormDescriptionConverterSwitch(interpreter, this.editService, this.objectService);
+
+        List<Switch<AbstractWidgetDescription>> widgetConverters = this.customWidgetConverterProviders.stream().map(provider -> provider.getWidgetConverter(interpreter, this.editService, this.objectService)).toList();
+        Collection<Switch<AbstractWidgetDescription>> switches = new ArrayList<>();
+        switches.add(new ViewFormDescriptionConverterSwitch(interpreter, this.editService, this.objectService, new ComposedSwitch<>(widgetConverters)));
+        switches.addAll(widgetConverters);
+        Switch<AbstractWidgetDescription> dispatcher = new ComposedSwitch<>(switches);
```

Comment:
since `ViewFormDescriptionConverterSwitch` already knows the `new ComposedSwitch<>(widgetConverters)`, may be the `Switch<AbstractWidgetDescription> dispatcher = new ComposedSwitch<>(switches);` is useless, so `dispatcher` should stay as it was before.

Reply:
Done.